.vscode/settings.json

```diff
@@ -1,6 +1,58 @@
 {
```

Comment:
99% sure this file shouldn't be in the repo.

Reply:
Mb, I'll add it to the gitignore